<launch>

<!--rosbag#####################################################################################################-->
    <param name="/use_sim_time" value="true" />
    <node pkg="rosbag" type="play" name="rosbag" args="-l /home/luo/rosframe_ws/src/pedol/bag/lRGB_dynamic11105hz.bag" />

<!--pedol######################################################################################################-->


    <!-- Manager -->
    <node pkg="nodelet" type="nodelet" args="manager" name="Mg" output="screen"  >
        
    </node>

    

    <!-- TDTrackingNodeletClass -->
    <node pkg="nodelet" type="nodelet" args="load pedol/TDTrackingNodeletClass Mg" name="TDTrackingNodeletClass" output="screen" >
        <param name="AgentId"        type="int"    value="0"/>
        <param name="AgentFrameId"   type="string" value="livox_frame" /> 
        <remap from="/TDTrackingNodeletClass/in" to="/livox/lidar" />
    </node>


    


<!-- rviz####################################################################################################### -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find pedol)/launch/rviz/pedol_visual.rviz" />

</launch>
